docs/modules/cloudmodule/controller.md

```diff
@@ -0,0 +1,55 @@
+# Edge Controller
+
+
+## Edge Controller Overview
+ controller(also known as edgecontroller) is the bridge between EdgeMaster(Kubernetes Api Server) and Edgecore
+
+   
+## Operations Performed By Edge Controller
+ 
+ The following are the functions performed by Edge controller :-
+  - Register and Start Sub Modules
+  - Downstream Controller-Sync add/update/delete event to edgeCore from EdgeMaster
```

Comment:
Replace EdgeMaster to Api-Server everywhere please

Reply:
Follow this standard while writing explanation everywhere to keep consistency. Dont use '-'
```
FOO: BAR.....
```